src/app/InteractionModelTimeout.h

```diff
@@ -22,7 +22,11 @@
 namespace chip {
 namespace app {
 
+#ifdef CONFIG_IM_BUILD_FOR_UNIT_TEST
 static constexpr System::Clock::Timeout kImMessageTimeout = System::Clock::Seconds16(12);
+#else
+static constexpr System::Clock::Timeout kImMessageTimeout = System::Clock::Seconds16(12);
```

Comment:
This is the same as the "test" value.  Why is this being added?

Reply:
Added by mistake, removed.